src/app/core/create-store.js

```diff
@@ -0,0 +1,29 @@
+
+/**
+ * Dependencies.
+ */
+
+var createStore = require('fluxible/utils/createStore');
+var extend = require('extend');
+
+/**
+ * Generic hydrate and dehydrate methods.
+ */
+
+var genericHydration = {
+  rehydrate: function (state) {
+    this.state = state;
+  },
+  dehydrate: function () {
+    return this.state;
+  }
+};
+
+/**
+ * An extension of fluxible's createStore method.
+ */
+
+module.exports = function (storeSpec) {
+  extend(false, storeSpec, genericHydration);
```

Comment:
Cool. Less ugly. Yeah, I didn't notice the optional nature of the first argument. An optional first argument kinda throws me of a bit! Yeah, I was explicitly saying "not a deep copy".